resource/tudo/support/css/tudo-index.css

```diff
@@ -279,6 +279,47 @@ p {
   margin-bottom: 1rem;
 }
 
+a,
+button,
+input,
+select,
+h2,
+tr,
+summary,
+textarea,
+span.progress,
+div.icons {
+  transition: outline-offset 0.1s ease !important;
+  outline-offset: 30px;
+  &:focus-visible {
+    outline: solid 3px var(--orangeish) !important;
+    outline-offset: -1px !important;
+  }
+  &:active {
+    outline-offset: -3px !important;
+  }
+}
```

Comment:
make sure focus stuff is there only once
if we remove flying-focus, this css should also be in other resource packs